testkit/src/main/scala/akka/kafka/testkit/KafkaTestkitTestcontainersSettings.scala

```diff
@@ -8,17 +8,18 @@ package akka.kafka.testkit
 import java.util.function.Consumer
 
 import akka.actor.ActorSystem
+import akka.kafka.testkit.internal.AlpakkaKafkaContainer
 import com.typesafe.config.Config
-import org.testcontainers.containers.{GenericContainer, KafkaContainer}
+import org.testcontainers.containers.GenericContainer
 
 final class KafkaTestkitTestcontainersSettings private (
     val confluentPlatformVersion: String,
     val numBrokers: Int,
     val internalTopicsReplicationFactor: Int,
-    val configureKafka: Vector[KafkaContainer] => Unit = _ => (),
-    val configureKafkaConsumer: java.util.function.Consumer[java.util.Collection[KafkaContainer]] =
-      new Consumer[java.util.Collection[KafkaContainer]]() {
-        override def accept(arg: java.util.Collection[KafkaContainer]): Unit = ()
+    val configureKafka: Vector[AlpakkaKafkaContainer] => Unit = _ => (),
+    val configureKafkaConsumer: java.util.function.Consumer[java.util.Collection[AlpakkaKafkaContainer]] =
+      new Consumer[java.util.Collection[AlpakkaKafkaContainer]]() {
+        override def accept(arg: java.util.Collection[AlpakkaKafkaContainer]): Unit = ()
```

Comment:
This is public API exposing internal classes. Can these be marked as `@InternalApi`?

Reply:
On `AlpakkaKafkaContainer`?